ws.ts

```diff
@@ -32,6 +39,10 @@ export const ws_handler = (
         get_events_by_filter: func_GetEventsByFilter;
         write_regular_event: func_WriteRegularEvent;
         write_replaceable_event: func_WriteReplaceableEvent;
+        // invitation
+        get_invite_to_space_event: func_GetInviteToSpaceByID;
+        accept_invitation: func_AcceptInvitation;
+        count_invite_to_space_each_invite: func_CountInviteToSpaceEachInvite;
```

Comment:
What purpose does it serve?

Reply:
Used to obtain the number of accept events associated with the same invite event in the DB to control invitations with quantity limits.